M/mlpack/build_tarballs.jl

```diff
@@ -68,6 +68,11 @@ elif [[ $target == *mingw* ]]; then
     fi
     FLAGS+=(-DLAPACK_LIBRARY=${libdir}/libopenblas${arch_ext}.${dlext})
     FLAGS+=(-DBLAS_LIBRARY=${libdir}/libopenblas${arch_ext}.${dlext})
+
+    # The Boost libraries on Windows appear to have been compiled with
+    # _GLIBCXX_USE_CXX_ABI=0, so we do the same here.
+    # https://stackoverflow.com/questions/33394934/converting-std-cxx11string-to-stdstring
+    FLAGS+=(-D_GLIBCXX_USE_CXX11_ABI=0)
```

Comment:
That shouldn't be a problem, we're expanding the C++ string ABI for this reason :confused:

Reply:
I've cancelled the jobs, we need to rebuild boost :disappointed:

Reply:
See #958.  After that is merged and registered, you should just remove this block.

Reply:
:+1: that sounds good, I'll keep an eye on that.  Thank you for digging into this!